src/src-schemas/subschemas/PID.json

```diff
@@ -0,0 +1,126 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "description": "PID segment",
+    "id": "/PID",
+    "type": "object",
+    "properties": {
+            "PID": {
+            "title": "Person identification",
+            "edi_tag": "PID",
+            "type": "object",
+            "properties": {
+                "qualifier": {
+                    "title": "party qualifier",
+                    "edi_ref": "3035",
+                    "edi_order": 1,
+                    "type": "string",
+                    "maxLength": 3,
+                    "enum": [
+                        "PAT"
+                    ]
+                },
+                "identification": {
+                    "title": "party identification",
+                    "edi_ref": "C082",
+                    "edi_order": 2,
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "title": "party identification, coded",
+                            "edi_ref": "3039",
+                            "edi_order": 1,
+                            "type": "string",
+                            "maxLength": 17
+                        },
+                        "qualifier": {
+                            "title": "code list qualifier",
+                            "edi_ref": "1131",
+                            "edi_order": 2,
+                            "type": "string",
+                            "maxLength": 3,
+                            "enum": [
+                                "CMS",
+                                "PCL"
+                            ]
+                        },
+                        "agency": {
+                            "title": "code list resp. agency, coded",
+                            "edi_ref": "3055",
+                            "edi_order": 3,
+                            "type": "string",
+                            "maxLength": 3,
+                            "enum": [
+                                "VEK",
+                                "LOC"
+                            ]
+                        }
+                    }
+                },
+                "personName": {
+                    "title": "Person Name",
+                    "edi_ref": "C_HC05",
+                    "edi_order": 3,
+                    "type": "object",
+                    "properties": {
+                        "qualifier": {
+                            "title": "Name format qualifier",
+                            "edi_ref": "H009",
+                            "edi_order": 1,
+                            "type": "string",
+                            "maxLength": 3
+                        },
+                        "achternaam": {
+                            "title": "achternaam",
```

Comment:
Would you mind changing fields to English just to keep standard in the project? We should probably look to supporting different locales in future thou.

Reply:
Sure, I already did that, but this one must have slipped throug